testrunner/src/TestRunner.scala

```diff
@@ -80,6 +80,7 @@ object TestRunner {
   }
 
   def matchFingerprints(
+      frameworkName: String,
```

Comment:
This breaks binary compatibility. Can you provide a binary compatible forwarder `def` without the `frameworkName` parameter, which delegates to this one with a fake framework name like `<unknown>`.